src/main/java/com/mycompany/worldofzuul/App.java

```diff
@@ -17,7 +17,7 @@
 
     @Override
     public void start(Stage stage) throws IOException {
-        scene = new Scene(loadFXML("menu/menu"));
+        scene = new Scene(loadFXML("rooms/lake"));
```

Comment:
Skal vel starte i menuen? ;)

Reply:
Ja, men det er for at slippe for at udfylde navn etc når vi stadig udvikler på det 🙂